src/nodes/configuration.h

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2025 Team Dissolve and contributors
+
+#pragma once
+
+#include "classes/configuration.h"
+#include "nodes/node.h"
+#include <memory>
+
+class ConfigurationNode : public Node
+{
+    public:
+    ConfigurationNode(Graph *parentGraph);
+    ~ConfigurationNode() override = default;
+
+    public:
+    std::string_view type() const override;
+    std::string_view summary() const override;
+
+    /*
+     * Definition
+     */
+    private:
+    // Empty atomic configuration
```

Comment:
Small thing, but I would comment this differently as it won't be empty for long!
```suggestion
    // Configuration object
```